server/routes/auth/providers/oidc.js

```diff
@@ -105,7 +106,9 @@ if (OIDC_CLIENT_ID) {
               name: profile.name,
               email: profile.email,
               avatarUrl: profile.picture,
-              username: profile[OIDC_USERNAME_CLAIM],
+              // Claim name can be overriden using an env variable.
+              // Default is 'preferred_username' as per OIDC spec.
+              username: get(profile, OIDC_USERNAME_CLAIM),
```

Comment:
Should we include the default as third argument here?

Reply:
This makes sense but would mean that the env defaults will start to be scattered around the code file. Current placement was intended to keep them all together.

Reply:
Oh, I missed it at the top – ignore.

Reply:
Oh 🤣. Rolling back.